lib/laplace.cu

```diff
@@ -158,6 +157,7 @@ namespace quda
 #endif
       } else if (in.Nspin() == 4) {
 #ifdef GPU_WILSON_DIRAC
```

Comment:
Can this be replaced with `NSPIN4`? (Likewise for staggered and `NSPIN1` above)

Reply:
It was intentional I didn't do this, and just have Laplace enabled by Wilson / staggered, as opposed to enabling it any generic nspin-4 or nspin-1 operator is enabled.  Alternative is to make a Cmake variable for this specifically.  Not sure I have strong feelings either way.